app/presenters/organisations/show_presenter.rb

```diff
@@ -148,6 +178,19 @@ def search_results_to_documents(search_results)
       }
     end
 
+    def contact_line(line)
+      return line + "<br/>" if line && line.length.positive?
+      ""
+    end
+
+    def make_email_link(email)
+      "<a href=\"#{email}\" class=\"brand__color\">#{email}</a>"
```

Comment:
Can we use `link_to` here instead?

Reply:
No. Actually, I just realised, we should be using `mail_to` otherwise these links don't work! Whoops.